app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -652,43 +652,73 @@ class AdvancedFilter extends React.Component {
    */
   getRelativeTooltipString(filter, value) {
     const filterName = filter.title.replace(' (Relative Date)', '');
-    let statement, rangeString, start, end;
+    let rangeString, before, after;
+    let statement = '';
+    const operatorValue = value.operator.replace('-', ' ');
 
-    // set variables for date options including a time stamp
-    if (filter.hasTimestamp) {
+    if (value.operator === 'more-than') {
       if (value.when === 'past') {
-        rangeString = 'dated through today’s date';
-        start = moment()
+        before = moment()
           .subtract(value.number, value.unit)
+          .add(1, 'd')
           .format('MM/DD/YY');
-        end = 'now';
       } else {
-        rangeString = 'with today’s date as of the current time';
-        start = 'now';
-        end = moment()
+        after = moment()
           .add(value.number, value.unit)
+          .subtract(1, 'd')
           .format('MM/DD/YY');
       }
+    } else if (value.operator === 'less-than') {
```

Comment:
I think we might actually want to just leave this as `else` instead of `else if` if we don't want to have to deal with a migration right now to handle saved filters that users already have in production for this

Reply:
This is a good point about the logic in general (not just the tooltip).  @holmesie @kylietmo do you think a migration is necessary for this? Should just be changing all the existing saved filters to the 'less-than' value option

Reply:
Ah good catch @timwongj. Yeah I think we should do a migration.

Reply:
migration is done, feel free to test

Reply:
This looks good Tim.  One weird thing I noticed is that we no longer support in the future for date fields with timestamps (enrolled, latest report) so those are converted to in the past in the migration.  Is this the right solution?  Should we remove them? I'm not sure

Reply:
hmm I'm not sure if the migration does that? all it does is:
- adds `less-than` as `operator` to all existing relative date option fields (because all the ones prior to this PR are `less-than`)
- adds `hasTimestamp` to all existing relative date option fields and populates it with the correct value

Reply:
Hmm I was definitely seeing it behave that way... maybe it won't be a problem because the PO team said those dates can't be in the future.  I can ask though

Reply:
@timwongj talked to the PO team, can we also add changing "in the future" relative date filters for timestamped date fields to "past"

Reply:
For record keeping, we removed the "in the future" relative date filters instead of doing the change above